numba/core/typing/templates.py

```diff
@@ -772,7 +781,17 @@ def _build_impl(self, cache_key, args, kws):
         jitter = self._get_jit_decorator()
 
         # Get the overload implementation for the given types
-        ovf_result = self._overload_func(*args, **kws)
+        try:
+            ov_sig = inspect.signature(self._overload_func)
```

Comment:
Should this be moved out of the try block as above?

Reply:
yes, thanks!